agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4781,6 +4782,23 @@ bool slave_thread::handle_ap_capability_query(Socket *sd, ieee1905_1::CmduMessag
     return true;
 }
 
+bool slave_thread::handle_multi_ap_policy_config_request(Socket *sd,
+                                                         ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    // TODO: This is a stub handler for the purpose of controller certification testing, will be
+    // implemented later on agent certification.
+    auto mid = cmdu_rx.getMessageId();
+    LOG(DEBUG) << "Received MULTI_AP_POLICY_CONFIG_REQUEST_MESSAGE, mid=" << std::hex << int(mid);
+
+    // send ACK_MESSAGE back to the controller
+    if (!cmdu_tx.create(mid, ieee1905_1::eMessageType::ACK_MESSAGE)) {
+        LOG(ERROR) << "cmdu creation of type ACK_MESSAGE, has failed";
```

Comment:
Add MID for the error log so we know which ACK was lost.

Reply:
The MID is being printed one line before when we receive the policy config request. I don't see the point to print it again.